meta-mender-core/recipes-mender/mender-configure/mender-configure.inc

```diff
@@ -44,6 +45,15 @@ do_install() {
     install -d -m 755 ${D}/data/mender-configure
     install -d -m 755 ${D}/var/lib/
     ln -s /data/mender-configure ${D}/var/lib/mender-configure
+
+    # For demo
+    cat > ${D}/data/mender-configure/device-config.json <<EOF
+{
+  "timezone": "UTC"
```

Comment:
Is UTC hardcoded? For my learning, isn't it something you can customize in Yocto?

Reply:
I also thought so, but all I could find was [this variable](https://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#var-INSTALL_TIMEZONE_FILE), which determines whether you want a timezone file installed or not. But it doesn't say anything about the content. I assume that what they want users to do is to set that variable to 0, and then install a file themselves. Unfortunately, since this can be done in any recipe, it's hard for us to pick it up at build time. If this was a production feature, I would have dug deeper into this, but I can't see a smarter solution by just scratching the surface.